api/pom.xml

```diff
@@ -97,10 +97,33 @@
          <artifactId>org.jacoco.core</artifactId>
          <scope>test</scope>
       </dependency>
+
+      <!-- Testing -->
+      <dependency>
+         <groupId>junit</groupId>
+         <artifactId>junit</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>org.assertj</groupId>
+         <artifactId>assertj-core</artifactId>
+      </dependency>
+
    </dependencies>
    <build>
       <plugins>
          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+               <forkCount>1C</forkCount>
+               <reuseForks>true</reuseForks>
+               <redirectTestOutputToFile>false</redirectTestOutputToFile>
+               <systemPropertyVariables>
+                  <log4j.skipJansi>true</log4j.skipJansi>
+               </systemPropertyVariables>
```

Comment:
I think that plugin configuration is not inherited from the parent pom. Moreover, I changed the values to run tests in multiple threads at the same time.